docs/rfcs/1000-shamir-based-recovery.md

```diff
@@ -726,6 +727,121 @@ The claimer gets access to `reveal_token` and `data_key`, it can then retrieve `
+        ],
+        "req": {
+            "cmd": "shamir_recovery_deletion",
+            "fields": [
+                {
+                    "name": "certificate",
+                    // Contains a ShamirRecoveryDeletionCertificate
+                    "type": "bytes"
+                }
+            ]
+        },
+        "reps": [
+            {
+                "status": "ok"
+            },
+            {
+                // Cannot deserialize data into the expected certificate
+                "status": "invalid_data"
+            },
+            {
+                // Given timestamp does not match previous timestamps
+                "status": "incoherent_previous_timestamp"
+            }
+            {
+                // Nothing to delete
+                "status": "no_setup_to_delete"
+            },
+            {
+                // Returned if the timestamp in the certificate is too far away compared
+                // to server clock.
+                "status": "timestamp_out_of_ballpark",
+                "fields": [
+                    {
+                        "name": "ballpark_client_early_offset",
+                        "type": "Float"
+                    },
+                    {
+                        "name": "ballpark_client_late_offset",
+                        "type": "Float"
+                    },
+                    {
+                        "name": "server_timestamp",
+                        "type": "DateTime"
+                    },
+                    {
+                        "name": "client_timestamp",
+                        "type": "DateTime"
+                    }
+                ]
+            },
+            {
+                // Returned if another certificate or vlob in the server has a timestamp
+                // posterior or equal to our current one.
+                "status": "require_greater_timestamp",
+                "fields": [
+                    {
+                        "name": "strictly_greater_than",
+                        "type": "DateTime"
+                    }
+                ]
+            }
+        ],
+    }
+]
+```
+The deletion certificate
+
+```json5
+{
+    "label": "ShamirRecoveryDeletionCertificate",
+    "type": "shamir_recovery_deletion_certificate",
+    "other_fields": [
+        {
+            "name": "author",
+            "type": "DeviceID"
+        },
+        {
+            "name": "timestamp",
+            "type": "DateTime"
+        },
+        {
+            "name": "previous_timestamp",
+            "type": "DateTime"
+        },
+        {
+            "name": "share_recipients",
+            "type": "List<UserId>"
+        }
+    ]
+}
+```
+
+The certificate needs to include the previous certificate timestamp in the deletion certificateto link both certificates together.
+
+> Future evolution: link the certificates with 1/ the brief signature 2/ a certificate id. This kind of link can be done between the share and brief certificate too.
```

Comment:
This is no longer needed if we have the user ID + timestamp (and is complex to add later on anyway given we have to support backward compatibility and hence not all certificates will have the ID...)

So I would suggest to just remove this comment